Clean Icon Names

Foggalong opened the pull request:
This continues the work of @bilelmoussaoui and finally fixes #3701. Switches out the coded names that Chrome and Steam use for descriptive ones and neatens names in a few other places.

Foggalong committed:
Changes from #4138

bilelmoussaoui commented on data.json:
```diff
@@ -741,14 +751,18 @@
             "root": "anydo",
             "symlinks": [
                 "any_do",
+                "chrome-ocgddccilgpeepgglnlpchkpgamkgmld-Default"
```
You have forgot a comma here

Foggalong replied:
I'm so glad you pointed that out, I was looking at the Travis output scratching my head as to what all the errors were 😅

Foggalong committed:
Missing comma 🙈

Foggalong committed:
Descriptive names for Steam games

Foggalong committed:
Sneaky s

Foggalong committed:
Update data.json

Foggalong committed:
DB Ordering

Foggalong commented:
That was a bit round the houses but we got there! I'll squash merge at the end :joy: But yeah, Steams games are now all descriptive. Fixed the issue with duplicates for `crawl` and `ricochet` by renaming both the originals.

Foggalong committed:
Descriptive names for Chrome apps

Foggalong committed:
Fix cut-the-rope position

Foggalong commented:
Chrome apps done! As a point of reference [this site](https://crx.dam.io/) which maintains a near complete [list of all chrome apps ever](https://crx.dam.io/ext/) and [this database of Steam apps](https://steamdb.info/search/?a=app&q=) proved really useful for unobfuscating icon names.

Foggalong committed:
Renamed last unity-webapps-* icon

Foggalong committed:
Switched Vbox to virtualbox

Foggalong committed:
Better filename for Origin (WINE app)

Foggalong commented:
Think we're all good now @bilelmoussaoui @palob :smile:

bilelmoussaoui commented:
Looks good to me. There are still few camel case icon names. Shouldn't be renamed too? Especially the AdobeXXX.svg icons

Foggalong commented:
Oh yeah, will add an extra commit for those!

*Edit: make that several commits. There are way more of these than I thought.*

Foggalong committed:
Start on CamelCase removal

Foggalong committed:
Comma correction

Foggalong committed:
Ordering correction & gplanarity
So turns out the gPlanarity people decided to call their icon gPicon. I
spent more time than I should have done looking for an app called
gPicon. I hate it when devs do this.

Foggalong committed:
Finishes CamelCase removal
Also changes filenames which contained ., _, or +

Foggalong committed:
Fix ordering and removed dud atav file

palob commented:
Are you sure no symlinks fell by the wayside or should I scrutinise thouroughly?

Foggalong commented:
I don't think I've missed anything, though to be honest it's going to difficult to check without going through the data.json changes line by line because new symlinks added and duplicate icons were removed. Checking commit by commit might be easier because in theory they don't have any overlap icon wise. If it looks like it'll be too fiddly to do that let me know and I can split this merge up into separate PRs for Steam, Chrome, and CamelCase :smile:

palob commented:
I'll just do a diff of the resulting theme release comparing to the previous release.

palob commented:
By the way, for an extra dose of consistency: ``liferea`` could be ``rss-blue``.

Foggalong committed:
Switched liferea to RSS Blue
This also removes the liferea icon for Shine and adds the RSS icon (and
all its symlinks) for uTouch

bilelmoussaoui reviewed (commented):
Why the  'shine' icon was removed?

Foggalong commented:
@bilelmoussaoui It was a duplicate of the normal RSS icon but I'm not adding new icons to Shine at the moment. I'd sooner it lose an icon for now and a future maintainer see that Liferea needs a new icon. It's worth noting that through the re-jiggling of names Shine & uTouch did get a load of new icons from these clean names :smile: 

As an aside, discrepancies life this could potentially be fixed with symlinks but again not gonna look too much into that myself whilever uTouch and Shine aren't supported.

palob commented:
``internet-mail`` and ``Adobeillustrator`` (hardcoded anyway, right?) are missing in the latest release.
Dunno if needed.

Foggalong commented:
`Adobeillustrator` isn't needed, that's a typo from way back when I originally added these icons. The correct icon name is `AdobeIllustrator` which is not included instead.

`internet-mail` should be included though - I accidentally set `mail_generic` as both the root icon and a symlink rather than `internet-mail` as the root as I'd intended.

Thanks for catching that, will submit a quick fix for it now!
